ee/tabby-webserver/Cargo.toml

```diff
@@ -29,8 +30,11 @@ thiserror.workspace = true
 tokio = { workspace = true, features = ["fs"] }
 tokio-rusqlite = "0.4.0"
 tokio-tungstenite = "0.20.1"
+tower = { version = "0.4", features = ["util"] }
```

Comment:
is this necessary?

Reply:
The `oneshot` method used in `resolve_file` method comes from `tower::ServiceExt` trait, so it's necessary.